sdk/python/core/tests/test_sanity_path.py

```diff
@@ -128,6 +128,21 @@ def test_get_schema(self):
         xml = self.codec.encode(res, EncodingFormat.XML, False)
         self.assertNotEqual( len(xml), 0 )
 
+    def test_anyxml(self):
```

Comment:
Could you please also add a testcase in test_sanity_codec.py ?

Reply:
I did...